ci/pipelines/deploy-to-test.yml

```diff
@@ -1571,6 +1572,31 @@ jobs:
         params:
           APP_NAME: toolbox
           <<: *wait_for_deploy_params
+      - task: run-toolbox-metric-tests
```

Comment:
This task wants to go into the run-integration-tests job rather than the deploy-toolbox job. On that job you can specify adot-candidate requiring deploy-toolbox to have passed

Reply:
Is that right? I only see that job in the PR pipeline.

Reply:
Or do you mean add a new `run-integration-tests` job to `deploy-to-test`?

Reply:
Sorry I mean [run-adot-integration-test](https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/deploy-to-test/jobs/run-adot-integration-test/builds/6)